crates/consensus/src/receipt/mod.rs

```diff
@@ -37,6 +37,7 @@ mod tests {
     use alloy_eips::eip2718::Encodable2718;
     use alloy_primitives::{address, b256, bytes, hex, Bytes, LogData};
     use alloy_rlp::{Decodable, Encodable};
+    use std::vec;
```

Comment:
this seems out of place. does it not cause redundant import lints?

Reply:
Ah good spot - not necessary, removed.